src/sentry/models/prompts_activity.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.conf import settings
+from django.utils import timezone
+from jsonfield import JSONField
+
+from sentry.db.models import (BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr)
+
+
+class PromptsActivity(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
```

Comment:
@mattrobenolt is there a rule of thumb for when we should use an FK vs not

Reply:
Not really a rule I can articulate. I think you should use a real foreignkey here though. The reasons we don't use them in places is for scaling when we need to split tables out onto different hosts. In which case, a foreign key doesn't work.

Right now, I don't see any issue with what you have, and we wouldn't be splitting the organization table out anytime soon. Same with project, you'll be fine.